### ISSUE TITLE
Any plans to finalize to general release?

### ISSUE THREAD

**Issue by jrose323** [gggeek/polyfill-xmlrpc#2]
This project has not seen any updates in over a year. Is there any plan to finalize this into general release version?

**Comment by gggeek** [comment]
Hello.

In over a year, there have been neither bugs or feature-requests filed, nor pull-requests opened. I am not sure if this is a sign of good code quality, or of low adoption rate :-D

I do have an internal "to do" list of improvements I was thinking to implement before tagging it as 1.0 final, but a few of those are better implemented in the upstream php-xmlrpc lib and, while I have been working on that one quite steadily since November, there are other changes which got prioritised (and I was in fact planning to work on the php-jsonrpc package next, before putting focus back on this one).

Long story short: do you have compelling reasons for not using version rc1? Regardless of its name, I do consider it quite good (stable) as it is...

**Comment by jrose323** [comment]
I am currently using it thus far without issue. My inquiry was more out of curiosity.

**Comment by gggeek** [comment]
Closing this, as 1.0 final is now out.
There finally was one issue found a pull request merged :-)
